packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -111,6 +111,11 @@ _Props/Attrs_
     [isEmpty](https://tiptap.dev/api/editor#is-empty) API. The component and the Angular directive will have a getter method
     that can be used to bind it in the Angular application.
 -   `fitToContent` - is a boolean attribute allows the text area to expand vertically to fit the content.
+-   `maxlength` - is a number attribute that restricts the number of input characters in the editor. This is achieved through
```

Comment:
It'd be good to mention that the name `maxlength` matches attributes on native elements (could even link to MDN docs for reference). I was initially confused why we didn't choose something more inline with our other APIs like `max-length` or `maximum-length`. But given there is precendence with native elements, I like the idea of following that precedence as you're currently suggesting.